Hogajama/hogajama-domain/src/main/java/com/gepardec/hogarama/domain/unitmanagement/entity/EntityNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package com.gepardec.hogarama.domain.unitmanagement.entity;
+
+public class EntityNotFoundException  extends RuntimeException{
+    private static final long serialVersionUID = 1L;
+
+    public EntityNotFoundException(Long sensorId, Class<?> clazz){
```

Comment:
Just a small issue - better rename parameter sensorId to entityId or id as this exception is a general one